atd-road-conditions/scrape.py

```diff
@@ -41,16 +41,28 @@
 def get_sensor_records():
     logger.debug("Getting sensors from Knack...")
     app = knackpy.App(app_id=KNACK_APP_ID, api_key=KNACK_API_KEY)
-    return app.get(KNACK_OBJECT, filters=KNACK_RECORD_FILTERS)
+    return app.get(KNACK_VIEW, filters=KNACK_RECORD_FILTERS)
 
 
 def create_sensor(record, session):
     ip = record.get("field_3595")
     sensor_id = record.get("field_3598")
-    if not ip and sensor_id:
+    lon = record.get("field_182", {}).get("longitude")
+    lat = record.get("field_182", {}).get("latitude")
+    location_name = record.get("field_211").strip()
```

Comment:
Good point Frank, it sounds like you are describing something similar to our config/knack.py file in atd-knack-services. `field_182` is so often the location field we are looking for in data_tracker, I've apparently internalized that to mean "location"

Reply:
lol yes `field_182` is everywhere. good point Frank—i'll move this and other settings into a separate file. i don't often fuss with env settings in Knack, because knack dev environments typically need to match exactly the production env.